escrow/free_call_storage.go

```diff
@@ -0,0 +1,53 @@
+package escrow
+
+import (
+	"reflect"
+)
+
+type FreeCallUserStorage struct {
+	delegate TypedAtomicStorage
+}
+
+func NewFreeCallUserStorage(atomicStorage AtomicStorage) *FreeCallUserStorage {
+	return &FreeCallUserStorage{
+		delegate: &TypedAtomicStorageImpl{
+			atomicStorage: &PrefixedAtomicStorage{
+				delegate:  atomicStorage,
```

Comment:
I just found that it means we cannot use MPE contract address as part of environment id in https://github.com/singnet/snet-daemon/pull/442. Network_id should be prefix of the AtomicStorage and we should add MPESpecific storage which has MPE address prefix.

Reply:
Ok, we will handle this in the other PR ..and standardize this